pubspec.yaml

```diff
@@ -13,7 +13,9 @@ dependencies:
   font_awesome_flutter: ^8.5.0
   provider: ^4.0.2
   transformer_page_view: ^0.1.6
-  flushbar: ^1.9.1
```

Comment:
The author seems to have not released the package for a long time,So I can only point to the git repository